content/css/concepts/display/terms/clear/clear.md

```diff
@@ -1,17 +1,17 @@
 ---
-Title: "clear"
-Description: "Species whether an element coming after a floated element should be moved down or not."
+Title: 'clear'
+Description: 'Species whether an element coming after a floated element should be moved down or not.'
```

Comment:
Specifies?

Not related to the changes but maybe worth fixing the typo

Reply:
hehe yea good catch